src/NuGet.Clients/NuGet.CommandLine/Commands/SignCommand.cs

```diff
@@ -11,6 +11,7 @@
 using System.Text;
 using System.Threading.Tasks;
 using NuGet.Commands;
+using NuGet.Common;
```

Comment:
remove this?

Reply:
HashAlgorithmName moved to a different namespace. I thought it could go also at first :smile:

Reply:
This is where the :shufflefurtherparrot: emoji is appropriate then :D